.github/workflows/e2e.yml

```diff
@@ -0,0 +1,74 @@
+name: Operator e2e tests
+
+on: 
+  push:
+     branches: [main]
+  issue_comment:
+    types: [created]
+  workflow_dispatch: {}
+
+jobs:
+
+  build:
+    
+    if: | 
+     (github.event.issue.pull_request && github.event.comment.body == '/run-e2e') ||
+     github.event_name == 'push' ||
+     github.event_name == 'workflow_dispatch'
+
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16.2
+
+
+    - name: Install kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+        kubectl version --client
+    
+    - name: Install kubectx   
+      env:
+          KUBECTX_URL: https://github.com/ahmetb/kubectx/releases/download/v0.9.3/kubectx
+      run: |
+        curl -fsSLO $KUBECTX_URL
+        chmod +x kubectx
+        sudo install -o root -g root -m 0755 kubectx /usr/local/bin/kubectx
+        
+    - name: Install stern and Download vertica RPM package
+      env:
+          VERTICA_CE_URL: "https://vertica-community-edition-for-testing.s3.amazonaws.com/XCz9cp7m/vertica-10.1.1-0.x86_64.RHEL6.rpm"
+          STERN_URL: "https://github.com/stern/stern/releases/download/v1.19.0/stern_1.19.0_linux_amd64.tar.gz"
+      run: |
+        curl -fsSLO $STERN_URL 
+        tar zxvf stern_1.19.0_linux_amd64.tar.gz
+        sudo cp stern_1.19.0_linux_amd64/stern /usr/local/bin/
+        curl $VERTICA_CE_URL -o vertica-x86_64.RHEL6.latest.rpm
+      
+    - name: Run e2e tests
```

Comment:
Should we split this step?  Have a step that installs krew and one that runs the e2e test?

Reply:
Oh, just saw your comment about this.  So you did try to separate them but it didn't work.  You can disregard this for now.

Reply:
yes Each time I install krew on one step and try to use it on a different one I get a "unknown command error". Will continue to investigate but for now I have them on the same step